cmd/virt-launcher/virt-launcher.go

```diff
@@ -366,8 +368,14 @@ func main() {
 		}
 	}
 
+	keepAfterQemuChan := make(chan struct{})
+	log.Log.Infof("keepAfterQemu %v", *keepAfterQemu)
+	if !*keepAfterQemu {
+		close(keepAfterQemuChan)
+	}
+
 	if !*noFork {
-		exitCode, err := ForkAndMonitor(*containerDiskDir)
+		exitCode, err := ForkAndMonitor(*containerDiskDir, keepAfterQemuChan)
```

Comment:
instead of passing in the chan should we hang after we log `log.Log.Reason(err).Error("monitoring virt-launcher failed")` only in the case of error? If we are shutting down successfully do we still want to wait?

Reply:
Per David's idea, this change is not needed anymore.